lib/src/test/java/io/ably/lib/test/realtime/RealtimePresenceTest.java

```diff
@@ -1683,20 +1685,35 @@ public void realtime_presence_attach_implicit_subscribe_fail() throws AblyExcept
 		AblyRealtime ably = null;
```

Comment:
Is there a way in this test to confirm "if the channel is in or enters the FAILED state before the operation succeeds, **it will result in the listener not being registered**"